src/EventNtupleMaker_module.cc

```diff
@@ -325,9 +328,11 @@ namespace mu2e {
       FType _ftype = Unknown;
       std::vector<std::string> fitNames = {"Unknown", "LoopHelix","CentralHelix","KinematicLine"};
 
+      // for trigger branch:
+      bool firstEvent = true;
       // helper functions
       void fillEventInfo(const art::Event& event);
-      void fillTriggerBits(const art::Event& event,std::string const& process);
+      void fillTriggerBranch(const art::Event& event,std::string const& process, bool firstEvent, TTree* &_ntuple);
```

Comment:
I'm surprised you need to pass the ntuple by argument. It should be accessible from within fillTriggerBranch alreadysince it is a member variable of the class. Is there something I'm missing?

Reply:
there was a reason previously but now I see its not needed.